.github/workflows/main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Check out this repo
-      uses: actions/checkout@v2
+      uses: actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675
```

Comment:
This doesn't seem relevant to this change. Can you explain?

Reply:
I am still in the process of figuring it out. The reason I am doing it is to pass the github check. We have this issue before, but at that time, I pass the test by changing the checkout@v3 to checkout@v2 and add the permission:write. This time, the presubmit is blocked again. Haven't find a right solution yet.